src/Transport/Connectivity/NamespaceManagerAdapter.cs

```diff
@@ -113,9 +113,15 @@ public Task<TopicDescription> GetTopic(string path)
             return manager.GetTopicAsync(path);
         }
 
+        // TODO: delete this once NamespaceManagerAdapter is internalized
         public Task DeleteSubscriptionAsync(string topicPath, string subscriptionName)
         {
-            return manager.DeleteSubscriptionAsync(topicPath, subscriptionName);
+            return ((INamespaceManagerAbleToDeleteSubscriptions) this).DeleteSubscription(new SubscriptionDescription(topicPath, subscriptionName));
+        }
+
+        Task INamespaceManagerAbleToDeleteSubscriptions.DeleteSubscription(SubscriptionDescription subscriptionDescription)
```

Comment:
Note to reviewers: This is a trick to not expose the internal API method as a public method